helm_deploy/hmpps-breach-notice-api/values.yaml

```
@@ -32,6 +32,8 @@ generic-service:
   namespace_secrets:
     hmpps-breach-notice-api:
       APPINSIGHTS_INSTRUMENTATIONKEY: "APPINSIGHTS_INSTRUMENTATIONKEY"
+      HMPPS-AUTH_CLIENT-ID: CLIENT_CREDS_CLIENT_ID
+      HMPPS-AUTH-SECRET: CLIENT_CREDS_CLIENT_SECRET
     rds-instance-output:
```

This should enable access to the topic in the MOJ environments:
```suggestion
    hmpps-domain-events:
      HMPPS_SQS_TOPICS_HMPPSBREACHNOTICEPUBLISHTOPIC: topic_arn
    rds-instance-output:
```